### PR TITLE
Hotfox - pubmed -> pubmedcentral

### PR DESCRIPTION
**Pull request by fabianvf**
Now against master. @erinspace @chrisseto

### CODE REVIEW

File: scripts/rename.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    while len(page) > 0:
+        for doc in page:
+            try:
+                yield RawDocument({
+                    'doc': doc.doc,
+                    'docID': doc.docID,
+                    'source': target,
+                    'filetype': doc.filetype,
+                    'timestamps': doc.timestamps
+                })
```
Comment (fabianvf) [inline scripts/rename.py]: Could put a doc.delete() here, but I really didn't want to.

**Comment by fabianvf** [comment]
```curl -XDELETE <ES_URI>/share_v2/pubmed``` should remove the old pubmed completely from the frontend. Will have to look into what we want to do long-term with regards to the old records.

File: scrapi/harvesters/pubmedcentral.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+from __future__ import unicode_literals
+
+from scrapi.base import schemas
+from scrapi.base import helpers
+from scrapi.base import OAIHarvester
+
+
+def oai_extract_url_pubmedcentral(header):
+    PMC_ID = header.replace('oai:pubmedcentral.nih.gov:', '')
+    return 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC' + PMC_ID
```
Comment (fabianvf) [inline scrapi/harvesters/pubmedcentral.py]: does PMC_ID have a preceding slash?

Comment (erinspace) [inline scrapi/harvesters/pubmedcentral.py]: nope, URLS generated resolve as is

Comment (erinspace) [inline scrapi/harvesters/pubmedcentral.py]: for example: http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3656538/

Comment (fabianvf) [inline scrapi/harvesters/pubmedcentral.py]: oh, ok

**Comment by chrisseto** [comment]
Merging but follow up discussion is required.

**Pull request merged**